.clang-format

```diff
@@ -8,17 +9,17 @@ UseTab:       Never
 # If the line length exceeds 100, "ColumnLimit:  80" is used in scripts/line-length.py
 ColumnLimit:  0
 ---
+Language: Cpp
+Standard: c++17
 # Customize only those options that differ from the base style!
 # Dumping the options of the base style for comparison:
 # clang-format -style=google -dump-config > .clang-format_google
-Language:     Cpp
 AccessModifierOffset: -2
 AlignAfterOpenBracket: DontAlign
 AlignOperands: false
 AllowShortFunctionsOnASingleLine: None
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
-AlwaysBreakBeforeMultilineStrings: true
```

Comment:
Why did you change this?

Reply:
We only overwrite options that differ from the base-style. `AlwaysBreakBeforeMultilineStrings: true` is already part of the google style we inherit from.

Reply:
The commit message refers to the advice given in the comment a few lines above.